src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -98,6 +98,11 @@ public class Measure {
   private String model;
 
   private MeasureMetaData measureMetaData = new MeasureMetaData();
+  
+  @NotBlank(
+      groups = {ValidationOrder1.class},
+      message = "Version ID is required.")
+  private String versionId;
```

Comment:
I'm asking this because if we plan to generate this on the server side, create measure call might get blocked because of this validation

Reply:
also not sure if this is part of the metadata or measure model itself

Reply:
@adongare Thanks a lot Ashok, those are good questions! So this new ID will be for the measure with every version change. It will be generated from client side when a new measure is created, for measures previously generated, the current measure Id can be its version ID. When there is version action, a new version ID will be generated, and that will be the measure Id. (this part will be implemented later).

I do have changes in madie-layout for new measure and madie-measure for updating measure in terms of version ID.

Reply:
got it. thank you for the clarification!